### DISCUSSION TITLE
BigtableBackupOperator

### DISCUSSION THREAD

**Discussion by namita113** [apache/airflow#30789] [Ideas]
Need Big Table backup operator to take the Big Table backups 

**Use case / motivation**

**Related Issues**

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by mik-laj** [comment]
Are you planning to develop this operator or is it a request for someone else to do it?  I do not understand the intent of this report. Could you please write a little more?

**Comment by jaketf** [comment]
I think I can provide some more context here:
GCP PSO has a customer that wants `BigTableBackupOperator` so they can easily use airflow to schedule / manage [backups](https://cloud.google.com/bigtable/docs/backups).

This should be fairly straight forward
- Add method to BigTableHook to run a table backup ([python client docs](https://googleapis.dev/python/bigtable/latest/table.html#google.cloud.bigtable.table.Table.backup))
- Add an operator that uses this method

**Comment by jaketf** [comment]
I am happy to work on this.

**Comment by mik-laj** [comment]
@jaketf  I assigned you to this ticket.

**Comment by jaketf** [comment]
thank you I will have a PR next week.

**Comment by mik-laj** [comment]
@jaketf Do you need any help?  

PS. I miss Sasha. 🐈

**Comment by eladkal** [comment]
@jaketf are you working on this issue?

**Comment by jaketf** [comment]
@eladkal apologies for the delay on this. I have not been able to look at this in some time. I'm happy to let someone else take this up if there is interest, or close if there are not users actively interested in this feature.
